OpenInfraSummit2022/CollaborationOverCompetition.md

```diff
@@ -0,0 +1,26 @@
+# Collaboration over Competition - Fostering a community of CSPs
+
+## Abstract (1000 chars)
+
+The Sovereign Cloud Stack (SCS) is built upon the shoulders of giants: With proven technology such
+as OpenStack and Kubernetes the SCS project is building a modular software stack. SCS is much more
+than just another cloud stack - much rather SCS defines standards as part of a community process in
```

Comment:
much rather SCS gathers an open community to join forces and define common standards.